packages/fast-usdc/test/utils/chain-policies.test.ts

```diff
@@ -0,0 +1,44 @@
+/** @file test that chain policies match data from a requirements-gathering spreadsheet */
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { ChainPolicies } from '../../src/utils/chain-policies.js';
+import { type ChainPolicy } from '../../src/types.js';
+
+/** Spreadsheet header becomes a tuple type with column names used as tags */
```

Comment:
good trail. I won't have the spreadsheet so I'll assume you transcribed the values correctly.